README.md

```diff
@@ -230,6 +230,12 @@ sensuctl user change-password admin --current-password 'P@ssw0rd!' --new-passwor
 sensuctl configure -n --username admin --password 'changeme'
 ```
 
+The following operating systems have had support removed until sensu-go packages exist to support these operating systems:
```

Comment:
Please move this under the `### Support Platforms` section. btw, that's a type and should be `Supported Platforms`

Also change wording to 

`The following platforms will be supported again once packages are released.`

Reply:
Changes made